CORBA_DFRS/src/caio_replica/udp_parser/UdpParser.java

```diff
@@ -44,7 +44,7 @@ protected BookFlightReply bookFlight(BookFlightOperation bookFlightOperation) {
 		BookFlightReply bookFlightReply = null;
 		try{
 			String date = (new SimpleDateFormat("YYYY/mm/DD")).format((new SimpleDateFormat("mm/DD/YYYY").parse(bookFlightOperation.getDate())));
-			String[] dep_dest = (bookFlightOperation.getDestination().split("|"));
+			String[] dep_dest = (bookFlightOperation.getDestination().split("\\|"));
```

Comment:
Agreed, its how we have it in our replicas. This was actually causing a problem since it was "|" before which doesn't work. I can have everyone use the global.Constants now since we are in the same project.